app/app.component.ts

```diff
@@ -44,27 +45,28 @@ export class AppComponent implements AfterViewInit, OnInit {
         private nodeService: NodeService,
         private sshKeyService: SSHKeyService,
         private vmSizeService: VmSizeService) {
-        this.settingsService.init();
-        this.sshKeyService.init();
-        this.commandService.init();
-        this.vmSizeService.init();
-        this.adalService.init(adalConfig);
-        this.accountService.loadInitialData();
-
-        this.hasAccount = accountService.currentAccount.map((x) => Boolean(x));
-
-        Observable
-            .combineLatest(accountService.accountLoaded, settingsService.hasSettingsLoaded)
-            .subscribe((loadedArray) => {
-                this.isAppReady = loadedArray[0] && loadedArray[1];
+            this.autoscaleFormulaService.init();
```

Comment:
Can you add a blank line here instead of indenting. The auto formatter will put it back